src/styles/util/variables.scss

```diff
@@ -0,0 +1,5 @@
+$card-width: 200px;
+$card-margin-r: 48px;
+$container-padding: 40px;
+$container-width: 4* $card-width + 3 * $card-margin-r + 2 * $container-padding;
```

Comment:
Also, it's better to move amount of the card in the row to the separate variable